deps/libuv.version

```diff
@@ -1,2 +1,2 @@
-LIBUV_BRANCH=julia-uv0.11.26
-LIBUV_SHA1=030481e9d659fd46702ab747caf2cbbe19d537ba
+LIBUV_BRANCH=julia-v1.7.5
+LIBUV_SHA1=80fe49af0f19f82da811129220c9bdeb13b99ebb
```

Comment:
Right. But you need to do `make NO_GIT=1 -C deps configure-libuv` to generate the right checksums for no-git builds.